modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/utils/string.ts

```diff
@@ -72,6 +99,21 @@ export function replaceSpacesWithDash(str: string) {
 	return str.replace(/\s+/g, '-');
 }
 
+/**
+ * Ensures that the string is between curly braces, if not, adds it.
+ */
+export function stringBetweenCurlyBraces(str?: string | undefined) {
+	if (str && str?.[0] !== '{') {
```

Comment:
just `if(str?.[0] !== '{')` or  `if(str && str[0] !== '{')`. The `?` already checks if it is null

Reply:
I will keep it like `if(str && str[0] !== '{')` as it allows the user to delete the string completelly. With `if(str?.[0] !== '{') ` it would transform the empty string to {}.